src/raftstore/store/store.rs

```diff
@@ -2786,6 +2742,74 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             error!("{} register compact cf-lock tick err: {:?}", self.tag, e);
         }
     }
+
+    fn on_check_peer_stale_state_tick(&mut self, event_loop: &mut EventLoop<Self>) {
```

Comment:
why remove to another new tick?

Reply:
Because the original implementation checks it on every raft base tick(1s) when a node is isolated, it may have many leader missing regions, then we will get a lot of `leader missing` log. It's annoying and expensive, so I think we should relax the check.